### PR TITLE
BL-13218: Add overflow-wrap to content headers

### PR DESCRIPTION
**Pull request by DanielKChristouKainos**
+ Increased the AE name character count to 150.

# New PR ~ front end 💻 

## Notes

** Before submitting a PR for PRODUCTION in MTS, consider: ** - 

- This solution is documented and approved against ACs in the Jira ticket
- Changes are browser tested (remember to conduct/organise testing before a tagged release if not yet)
- Branch name include Jira ticket ref
- If helpful or material, add change to the Changelog.md file

** Style **

- CSS updates will not unwillingly affect other components - please highlight any risks
- Console logs and commented-out code is removed
- No inline JavaScript or CSS

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "dvsa/front-end",
   "description": "DVSA front-end assets used within all applications",
-  "version": "1.5.14",
+  "version": "1.4.5",
```
Comment (JoshuaLicense) [inline composer.json:4]: This doesn't look right

File: src/assets/scss/govuk-frontend/overrides/_overrides.scss [hunk 2]
```diff
@@ -196,4 +196,8 @@ $govuk-radios-focus-width: $govuk-focus-width + 1px;
     outline:3px solid $focus-colour;
 	outline-offset: 0;
 	background-color: $focus-colour;
-}
\ No newline at end of file
+}
+
+#businessName {
+    word-break: break-all;
+}
```
Comment (JoshuaLicense) [inline src/assets/scss/govuk-frontend/overrides/_overrides.scss:203]: I'm not sure the benefit of fixing it here - we'd have to do it on a much wider scale than just a single table cell.

I'd say take this out for now.

File: src/assets/scss/mts/modules/content-header/_content-header.scss [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
   &__title {
     @include bold-48;
+    word-break: break-all;
```
Comment (JoshuaLicense) [inline src/assets/scss/mts/modules/content-header/_content-header.scss:19]: Checking how other libraries handle this, the following seems to be the accepted pattern to break words.

```css
overflow-wrap: break-word;
word-wrap: break-word; // IE
```

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]

File: src/assets/scss/mts/modules/key-value-list/_key-value-list.scss [hunk 4]
```diff
@@ -24,6 +24,8 @@
   &__value {
     @include bold-19;
     padding-left: $gutter-half;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
```
Comment (JoshuaLicense) [inline src/assets/scss/mts/modules/key-value-list/_key-value-list.scss:28]: What elements use this class out of interest?

File: src/assets/scss/govuk-frontend/overrides/_overrides.scss (hunk 2 above)
Comment (DanielKChristouKainos) [reply src/assets/scss/govuk-frontend/overrides/_overrides.scss]: My concern was that doing it on a wider scale will affect cells (and other usages of cells in tables too potentially) beyond the scope of this ticket. I will push up a version that affects it on a wider scale but we will have to be aware of that.

File: src/assets/scss/mts/modules/key-value-list/_key-value-list.scss (hunk 4 above)
Comment (DanielKChristouKainos) [reply src/assets/scss/mts/modules/key-value-list/_key-value-list.scss:28]: By a quick glance it looks to be recording events, risk assessment and group tester tables.

**Review by CKindred — APPROVED** [review]

**Review by JoshuaLicense — CHANGES_REQUESTED** [review]
Can you double-check the dist file changes are expected please?

Are you expecting this rule to be in the dist:
image

**Comment by JoshuaLicense** [comment]
Are we expecting a `package.lock` change? 

There seem to be way more changes than I expected, could you double-check the diffs are expected, please?

**Comment by DanielKChristouKainos** [comment]
> Are we expecting a `package.lock` change?
> 
> There seem to be way more changes than I expected, could you double-check the diffs are expected, please?

Yes we are expecting a change since I did a minor version bump to reflect the minor change.

**Comment by JoshuaLicense** [comment]
I know there's a version bump - but a version bump shouldn't be ~26,300 lines.

image

**Comment by DanielKChristouKainos** [comment]
> I know there's a version bump - but a version bump shouldn't be ~26,300 lines.
> 
> image

Strange, I will checkout the change and generate it again. Since the only change in package.json is the version bump there should not be this many changes, unless it was out of date to begin with.

**Comment by DanielKChristouKainos** [comment]
OK I think I can see what is happening - there is a desync between the package.json and the package-lock.json .
On master I can see that the lockfile was generated using lockfile version 2 format:
https://raw.githubusercontent.com/dvsa/front-end/master/package-lock.json
Probably using npm 7:
https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json 
But we are targeting npm 6 which uses the old version 1 format:
https://github.com/dvsa/front-end/blob/19cdfb48eee9dc6eee0f0382ea9f3e593c3663e6/package.json#L34

**Comment by DanielKChristouKainos** [comment]
The lockfile in my branch is targeting version 1 BTW so that is correct if we want to stick to NPM 6.

**Review by JoshuaLicense — APPROVED** [review]

**Pull request merged**